database/__init__.py

```diff
@@ -57,6 +62,28 @@ def pragma_on_connect(dbapi_con, con_record):
 )
 
 
+def access_required(name):
```

Comment:
This looks like it should go into `authentication.py`

Reply:
Agreed. Feels free to move it, I’m on vacation without my laptop right now 😝